webui/react/src/components/Json.tsx

```diff
@@ -10,14 +10,14 @@ type TextTransfomer = (key: string) => string;
 interface Props {
   alternateBackground?: boolean;
   hideDivider?: boolean;
-  json: RawJson;
+  json: JsonObject;
   translateLabel?: TextTransfomer;
 }
 
 interface RowProps {
   label: string;
   translateLabel?: TextTransfomer;
-  value: RawJson | string | number | null;
+  value: Json | string | number | null;
```

Comment:
nit: `Json` includes `string | number | null`, so we could replace with
```suggestion
  value: Json;
```
alternatively, if we want to retain the same semantics as before, we should use `JsonObject`:
```suggestion
  value: JsonObject | string | number | null;
```

Reply:
Actually made this change then accidentally reverted it before committing >.>